scripts/linters/custom_eslint_checks/rules/protractor-practices.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
         ' "for loop" instead'),
       disallowAwait: 'Please do not use await for "{{propertyName}}()"',
       useProtractorTest: (
-        'Please use “.protractor-test-” prefix classname selector instead of ' +
+        'Please use “.e2e-test-” prefix classname selector instead of ' +
```

Comment:
Rename the key as well*

Also, I'm not sure whether at this stage we can remove all other "protractor" word from this file? (Note these are developer/user-facing messages.)

Reply:
Done. Yes, we need to change this as we will be following this naming rule instead of protractor-test. I will update the wiki as well after this PR will be merged.

Reply:
Also, as now this tests is generally for all e2e tests I have renamed the files to e2e-practices.js and also replaced all instance of protractor with e2e.